introduction.md

```diff
@@ -0,0 +1 @@
+hello my name is Angel Rodriguez my github user is angelfeliz and i am a developer
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.